compiler/src/dotty/tools/dotc/transform/LinkScala2Impls.scala

```diff
@@ -78,6 +78,12 @@ class LinkScala2Impls extends MiniPhase with IdentityDenotTransformer { thisPhas
         val impl = implMethod(sel.symbol)
         if (impl.exists) Apply(ref(impl), This(currentClass) :: args).withSpan(app.span)
         else app // could have been an abstract method in a trait linked to from a super constructor
+      case Apply(sel @ Select(Super(a, b), _), args)
```

Comment:
This part should be improved but basically tells to do the transform for constructors even on scala.js